package.json

```diff
@@ -20,6 +20,7 @@
     "angular-ui-bootstrap": "1.3.3",
     "angular-ui-router": "0.4.2",
     "bootstrap": "3.3.7",
+    "d3": "^4.9.1",
```

Comment:
metrics-graphics already has a dependency on d3, and I can't see any other direct usages of d3 from Perfherder itself, so this entry should be removed - could you open a followup?